### ISSUE TITLE
Add multiple Windows prefix locations to WineGUI

### ISSUE THREAD

**Issue by RichardJECooke** [winegui/WineGUI#24]
I have lots of different Windows folders in different places - .wine, .snap, flathub somewhere, Bottles, Steam.

How do I get WineGUI to see them all please?

**Comment by melroy89** [comment]
The default wine directory: `~/.wine` should already be visible in WineGUI by default.

Ideally try to create/place/move others as much as possible to folder: `<home_folder>/.local/share/winegui/prefixes`  directory (notice that `.local` is hidden). This is the default directory.

If you didn't create any Windows machines in WineGUI, you can also **consider changing the machine folder location** at: `File->Preferences` and select another folder location were most of your Windows Bottles/Machines are located.

Finally, if you know how to work on command line, you can also use the `ln -s` command to create so called symbolic link under Linux. For example: `ln -s /home/user/your-other-machine_folder /home/user/.local/share/winegui/prefixes/your-other-machine`

In this last example a symbolic link will be created between the existing machine and the WineGUI prefixes location, without the need of moving your Windows Machine/Bottle.

**Comment by RichardJECooke** [comment]
Thanks. But you can't move cdrives out of their app. Bottles/snap/flatpak all place them in very specific folders. Hopefully one day there'll be a settings page in winegui where you can add existing. cdrives

**Comment by melroy89** [comment]
> Thanks. But you can't move cdrives out of their app. Bottles/snap/flatpak all place them in very specific folders. Hopefully one day there'll be a settings page in winegui where you can add existing. cdrives

I understand, for now try to create a symbolic link via `ln -s`. I marked this issue has an enhancement now as well.

**Comment by melroy89** [comment]
Upstream ticket created: https://gitlab.melroy.org/melroy/winegui/-/issues/42

**Comment by RichardJECooke** [comment]
What does the machine folder do? You say ` consider changing the machine folder location`. I moved my entire wine drive to my storage drive (/mnt/storage) because it needs to be 500GB. I don't see how changing the machine folder will allow winegui to see it.

**Comment by melroy89** [comment]
The machine folder location it the place where all wine bottles are stored and read.

So if you move your bottles to your other disk (within a specific directory). 

You can change the machine folder location in winegui to point to a specific directory on that disk. From that moment forwards again the wine bottles are read and created on that new location.

**Comment by RichardJECooke** [comment]
Thanks. What is a bottle? Is that the same as a wine prefix?

But if my wine prefix is at `/mnt/storage/wine` and I make the machine folder location also `/mnt/storage/wine`, won't WineGUI pollute my Windows drive with all the WineGUI files? Should I make it the same folder?

**Comment by melroy89** [comment]
Yes bottle is the same as a wine prefix. And in winegui I just try to call it a windows machine to make it even easier to most people.

Yes you should create a directory on the top level indeed
Like /mnt/storage/prefixes. And you put all your prefixes within that prefixes directory.

**Comment by RichardJECooke** [comment]
Thanks again. But my question is since I already have the prefix, where should I point the machine folder to?

**Comment by melroy89** [comment]
I believe we went over this earlier. 

You can either move the prefix. Or use a symbolic link (using ln) or just create a new bottle or a copy of your bottle.
